server/index.js

```diff
@@ -1,13 +1,50 @@
 const path = require('path');
-
 const express = require('express');
+const helpers = require('./helpers');
 
 const app = express();
 const port = 3000;
 
 app.use(express.static(path.join(__dirname, '..', 'public')));
 app.use(express.json());
 
+//Products GET /products Retrieves the list of products
+app.get('/products', (req,res) => {
+  helpers.getProductsList()
+    .then((response) => {
+      res.send(response.data)
+    })
+    .catch((error) => {
+      console.log(error.data);
+    })
+});
+
+//Products GET /products/:product_id Returns all product level information for a specified product id
+app.get('/products/:product_id', (req, res) => {
+  helpers.getProductsList()
+  .then((response) => {
+    res.send(response.data)
+  })
+  .catch((error) => {
+    console.log(error.data);
+  })
+});
+
 app.listen(port, () => {
   console.log(`Example app listening at http://localhost:${port}`)
-});
\ No newline at end of file
+});
+
+
+/* gLearn notes */
+/*
+//[https://learn-2.galvanize.com/cohorts/2474/blocks/94/content_files/Front%20End%20Capstone/project-atelier-catwalk/products.md]
+//Atelier API can currently be found at https://app-hrsei-api.herokuapp.com/api/fec2/:CAMPUS_CODE/
+//In an HTTP GET request, parameters are sent as a query string: http://example.com/page?parameter=value&also=another
+//In an HTTP POST or PUT request, the parameters are not sent along with the URI, but in the request body.
+//...
+//  Products[https://learn-2.galvanize.com/cohorts/2474/blocks/94/content_files/Front%20End%20Capstone/project-atelier-catwalk/products.md]
+//  Reviews[https://learn-2.galvanize.com/cohorts/2474/blocks/94/content_files/Front%20End%20Capstone/project-atelier-catwalk/reviews.md]
+//  Questions&Answers[https://learn-2.galvanize.com/cohorts/2474/blocks/94/content_files/Front%20End%20Capstone/project-atelier-catwalk/qa.md]
+//  Cart[https://learn-2.galvanize.com/cohorts/2474/blocks/94/content_files/Front%20End%20Capstone/project-atelier-catwalk/cart.md]
+//  Interations[https://learn-2.galvanize.com/cohorts/2474/blocks/94/content_files/Front%20End%20Capstone/project-atelier-catwalk/interactions.md]
+*/
```

Comment:
These notes are useful for referencing during development, but not as useful for the users. To that end, I think we should probably delete them before we deploy the final product.